wavenet.py

```diff
@@ -20,39 +20,40 @@ def _create_dilation_layer(self, input_batch, layer_index, dilation):
         '''Adds a single causal dilated convolution layer.'''
 
         # The filter widths can be configured as a hyperparameter.
-        wf = tf.Variable(tf.truncated_normal(
-            [1, self.filter_width, 256, 256],
+        weights_filter = tf.Variable(tf.truncated_normal(
+            [1, self.filter_width, self.channels, self.channels],
```

Comment:
That is, assuming that at each layer we map N channels to N channels... Another option would be to give a vector with the same size as `dilations` specifying the number of channels for each layer.